tests/framework/config/cluster.go

```diff
@@ -36,6 +36,7 @@ type ClusterConfig struct {
 	StrictReconfigCheck bool
 	AuthToken           string
 	SnapshotCount       int
+	WithLearner         bool
```

Comment:
Learner should be a running server instance as one of the processes. We should make it explicit if by renaming it to  `ClusterSize` -> `MemberCount` and `WIthLearner` -> `LearnerCount`.